diff --git a/lib/sloppy.js b/lib/sloppy.js
index b5d8950a8ea98d2d58723c1f96eeabb260699e24..1e4999505ef13a428b87b3575d6f047dc976543d 100644
--- a/lib/sloppy.js
+++ b/lib/sloppy.js
@@ -6,19 +6,21 @@
 /* jshint -W085 */
 module.exports = {
   Window_run: function _run(code, file) {
-    if (file) code += '\n//@ sourceURL=' + file;
-    with(this) eval(code);
+    console.log('pnpm patch domino@2.1.6 sloppy.js')
+    // if (file) code += '\n//@ sourceURL=' + file;
+    // with(this) eval(code);
   },
   EventHandlerBuilder_build: function build() {
-    try {
-      with(this.document.defaultView || Object.create(null))
-        with(this.document)
-          with(this.form)
-            with(this.element)
-              return eval("(function(event){" + this.body + "})");
-    }
-    catch (err) {
-      return function() { throw err; };
-    }
+    console.log('pnpm patch domino@2.1.6 sloppy.js')
+    // try {
+    //   with(this.document.defaultView || Object.create(null))
+    //     with(this.document)
+    //       with(this.form)
+    //         with(this.element)
+    //           return eval("(function(event){" + this.body + "})");
+    // }
+    // catch (err) {
+    //   return function() { throw err; };
+    // }
   }
 };